src/app/shared/components/Container/PageTitleContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import { cn } from '@/lib/utils'
+import { FC } from 'react'
+import { CommonComponentProps } from '../commonProps'
+import { Typography } from '@/components/TypographyNew/Typography'
+
+export type PageTitleContainerProps =
+  | (CommonComponentProps & {
+      children: React.ReactNode
+      leftText?: never
+      rightText?: never
+    })
+  | (CommonComponentProps & {
+      children?: never
+      leftText: string
+      rightText?: string
+    })
+
+const ContainerCompnent: FC<PageTitleContainerProps> = ({ className = '', children }) => (
+  <div data-testid={'PageTitleContainer'} className={cn('flex w-full', className)}>
+    {children}
+  </div>
+)
+
+export const PageTitleContainer: FC<PageTitleContainerProps> = ({
+  leftText,
+  rightText,
+  className = '',
+  children,
+}) => {
+  if (children) {
+    return <ContainerCompnent {...{ className, children }} />
+  }
+
+  return (
+    <div data-testid={'PageTitleContainer'} className={cn('flex h-10 pr-[36.19rem] items-center', className)}>
+      <Typography as="h1" variant="h1" data-testid={'PageTitleContainerLeft'}>
+        {leftText}
+      </Typography>
+
+      <Typography
+        as="h2"
+        variant="h2"
+        className="grow uppercase text-right"
+        data-testid={'PageTitleContainerRight'}
+      >
+        {rightText}
+      </Typography>
+    </div>
+  )
+}
```

Comment:
The components you moved are not up-to-date with the [main](https://github.com/RootstockCollective/dao-frontend/pull/1014/files#diff-8b25d51df1e92aa5475da8910374ad07f9b7e6ed90d11c60b90397d212718770L1-L22)!